playbot/mind.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, log_method):
         """
         self.__log = log_method
         self.__moves = ['NOP()', 'PATCH()', 'SCAN()', 'OVERLOAD()', 'OVERHEAR()', 'EXPLOIT()', 'INFECT()']
-        self.__my_name = f'RandBot'
+        self.__my_name = f'RandBot_{randrange(1,100)}'
```

Comment:
w8.
I change this line three weeks ago and erased random number part, so it was possible to rank ELO for this kind of bot.
With random name evaluation of ELO is near to impossible.
My last change was based on the assumption that the same bot cannot participate in a fight against itself - from the point of view of ELO calculation such a game is meaningless.

Another thing: `randrage(1,100)` - why range is so small? Despite drawing a number, probability of the same name is high and will occur for a range of 99 options (on average once in 99 games - so often)

Before my change the draw range was 500, set by me :P